humanguide/app-layout/dialogs.rst

```diff
@@ -0,0 +1,22 @@
+Dialogs
+=======
+
+A dialog is a way of informing the user about important notices or letting them complete a single action.
+
+Dialogs should be unambiguous and concise in their wording. Only the most common, or main, action should be highlighted with color. The main action should be placed topmost in vertical dialogs or rightmost in horizontal dialogs.
+
+Main actions may be positive, negative, or neutral. Positive actions add something to their subjects. Negative actions remove, delete, or otherwise make changes that can't be easily undone. Neutral actions do not add or remove from the subject, but still make a change.
+
+The main action should be highlighted using ``theme.palette.normal.negative`` if it is destructive, ``theme.palette.normal.positive`` if it is positive, or ``theme.palette.normal.focus`` if it is neutral. The remaining options should be gray, without defining a color theme property.
+
+.. figure:: /_static/images/humanguide/40.png
```

Comment:
Could these examples use English translations instead? I know we can't replace them in other languages, but they will be more familiar (or at least understandable) to most users.